assembl/static2/js/app/components/common/fileUploader.jsx

```diff
@@ -73,17 +76,18 @@ class FileUploader extends React.Component {
         {this.props.withPreview
           ? <div className={this.state.fileSrc ? 'preview' : 'hidden'}>
             <img
-              src={this.state.fileSrc}
+              src={fileSrc}
               ref={(p) => {
                 this.preview = p;
               }}
-              alt="preview"
+              alt={fileIsImage || urlIsImage ? 'preview' : ''}
```

Comment:
we shouldn't render an img tag if this is not an image, aren't we ?

Reply:
this was made to satisfy a test (hence the comment made by Aryan up top), "preview" was displayed next to the file title when the uploaded file was not an image. The test mocks an update using an image url, so the condition isImage was false because it only checks if the file type is "image", which does not work with url. So I will get rid of this and update the test instead